### ISSUE TITLE
[v5] Uncaught TypeError: Cannot read property 'result' of undefined

### ISSUE THREAD

**Issue by farcaller** [welldone-software/why-did-you-render#150]
Forked off #149, as to not pollute it.

Given the same setup as in #149 and the following `App.tsx`:

```tsx
import React, { useState } from 'react';

function App() {
  const [whatever, setWhatever] = useState(1);

  const clickme = () => {
    setWhatever(whatever + 1);
  }

  return (
    <div>
      <button onClick={clickme}>test</button>
      <div>{whatever}</div>
      <Child/>
    </div>
  );
}

function Child() {
  return (
    <div>child</div>
  );
}

Child.whyDidYouRender = true

export default App;
```

With whyDidYouRender configured for `trackHooks: true`, so supposedly only tracking the state of `<Child/>`, the app crashes as soon as the state of `<App/>` changes (by clicking the button) with:

```
Uncaught TypeError: Cannot read property 'result' of undefined
    at getUpdateInfo.js:10
    at Array.map (<anonymous>)
    at V (getUpdateInfo.js:5)
    at B (getUpdateInfo.js:28)
    at G (getUpdateInfo.js:45)
    at Child (patchFunctionalOrStrComponent.js:21)
    at renderWithHooks (react-dom.development.js:14803)
    at updateFunctionComponent (react-dom.development.js:17034)
    at beginWork (react-dom.development.js:18610)
    at beginWork$1 (react-dom.development.js:23179)
    at performUnitOfWork (react-dom.development.js:22157)
    at workLoopSync (react-dom.development.js:22130)
    at performSyncWorkOnRoot (react-dom.development.js:21756)
    at react-dom.development.js:11089
    at unstable_runWithPriority (scheduler.development.js:653)
    at runWithPriority$1 (react-dom.development.js:11039)
    at flushSyncCallbackQueueImpl (react-dom.development.js:11084)
    at flushSyncCallbackQueue (react-dom.development.js:11072)
    at discreteUpdates$1 (react-dom.development.js:21893)
    at discreteUpdates (react-dom.development.js:806)
    at dispatchDiscreteEvent (react-dom.development.js:4168)
```

Looking deeper, `getOwnerDifferences` sees two hooks in `prevOwnerData`, which doesn't make sense to me as there's clearly a single hook in the source code.

![image](https://user-images.githubusercontent.com/693/93070840-4bb64900-f677-11ea-8ad9-9370509028d8.png)

**Comment by farcaller** [comment]
Repro'd only on WDYR 5.0.0-alpha.2, 4.3.1 works as intended.

**Comment by vzaidman** [comment]
thanks!
5.0.0-alpha.2 is indeed work in progress!

**Comment by vzaidman** [comment]
fixed in ` 5.0.0-alpha.4`
